python/cmake/ConfigureAlgorithmsHelpers.cmake

```diff
@@ -13,15 +13,26 @@
 # =============================================================================
 
 
+function(add_module_gpu_default FILENAME)
+    set (extra_args ${ARGN})
+    list(LENGTH extra_args extra_count)
+    if (${extra_count} GREATER 0 OR
+        ${CUML_UNIVERSAL} OR
+        ${CUML_GPU})
+      list(APPEND cython_sources
+           ${FILENAME})
+      set (cython_sources ${cython_sources} PARENT_SCOPE)
+    endif()
+endfunction()
```

Comment:
This function is used in ConfigureAlgorithms.cmake but also in linear_model/CMakeLists.txt. As more algorithms are added to the `CUML_ALGORITHMS`-controlled support, do you anticipate this being used in more CMakeLists files in the Cython builds? If so, the filename seems too narrow.